1.1.6/flake.nix

```diff
@@ -0,0 +1,44 @@
+{
+  inputs = {
+    # Pinned to the commit that updated Terraform.
+    nixpkgs.url = "github:nixos/nixpkgs/3c2757d25d1ea5133fe8d207e995032b722b9e39";
+  };
+
+  outputs = { self, nixpkgs, ... }:
+    let
+      lib = nixpkgs.lib;
+      pkgs = import nixpkgs {
+        system = "x86_64-linux";
+        # The AWS Session Manager plugin is unfree, so we add it to a whitelist.
+        config.allowUnfreePredicate = pkg: builtins.elem (lib.getName pkg) [
+          "ssm-session-manager-plugin"
+        ];
+      };
+    in {
+      defaultPackage."x86_64-linux" = pkgs.dockerTools.buildLayeredImage {
+        name = "ghcr.io/d3b-center/terraform";
+        tag = lib.removePrefix "terraform-" pkgs.terraform_1.name;
+
+        contents = with pkgs; [
```

Comment:
@rbreslow I really enjoy how minimalistic the docker container is. The packages included are only allowing to use of public modules since it won't be able to checkout git+ssh. Are we planning to keep this as a standard(only using public modules in Terraform and all further modules are public)?

Reply:
I agree. See [1fe1e40](https://github.com/d3b-center/docker-terraform/pull/1/commits/1fe1e4074e234f33521981135bcdf2ad74cbea21).